FlowCrypt/Functionality/DataManager/Encrypted Storage/KeyChainService.swift

```diff
@@ -19,23 +36,57 @@ actor KeyChainService {
     private let logger = Logger.nested("KeyChain")
     private let keyByteLen = 64
 
+    private let filesManager = FilesManager()
+    private let userDefaults = UserDefaults.standard
+
+    private enum Constants {
+        static let prefixUserDefaultsKey = "indexSecureKeychainPrefix"
```

Comment:
Let's call this `legacyKeychainIndexPrefixInUserDefaults` ? (`legacy` here means something we are moving away from)